packages/launchpad/src/components/code/InlineCodeEditor.spec.jsx

```diff
@@ -1,23 +1,25 @@
 import HeartIcon from 'virtual:vite-icons/mdi/heart'
 import InlineCodeEditor from './InlineCodeEditor.vue'
-import Input from '../input/Input.vue'
+import InputOld from '../input/InputOld.vue'
+import { ref } from 'vue'
 
 describe('InlineCodeEditor', () => {
   it('renders', () => {
+    const value = ref('console.log("hello world")')
+
     cy.mount(() => (
       <div>
-        <Input
+        <InputOld
```

Comment:
Is it possible to avoid having deprecated code in a greenfield codebase?

Is `InputOld` deprecated? or just old?

Reply:
Deprecated. We need to use the newer one but I haven't fixed the usages inside of InlineCodeEditor etc.